app/move/controllers/create/assessment.js

```diff
@@ -39,6 +38,8 @@ class AssessmentController extends CreateBaseController {
       showPreviousAssessment,
     } = req.form.options
 
+    const profileService = req.services.profile
```

Comment:
What are the thoughts on calling this directly on line 47 instead of creating the variable for it?

So we'd have:

```javascript
profile = await req.services.profile.create(person.id, {})
```